server/server_test.go

```diff
@@ -1535,12 +1568,25 @@ func TestServer_PublishAsJSON(t *testing.T) {
 	require.Equal(t, "google.pdf", m.Attachment.Name)
 	require.Equal(t, "http://ntfy.sh", m.Click)
 	require.Equal(t, "https://ntfy.sh/static/img/ntfy.png", m.Icon)
+	require.Equal(t, "", m.ContentType)
```

Comment:
since it's a new field, maybe we can have it set to `text/plain` when not markdown? makes it easier to do match against in the future if needed.

Reply:
I don't see the need to. The default has always been plaintext, and without a content type that's assumed. 99% of messages are plaintext, and adding the content type would just be useless bytes everywhere.